plugin-build/build.gradle.kts

```diff
@@ -70,3 +70,34 @@ if (gradle.gradleVersion >= "6.6.0") {
     val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
     publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
 }
+
```

Comment:
right now the gradle plugin is gpg signing the files, if we don't want it, we need to
https://github.com/getsentry/sentry-java/blob/main/build.gradle.kts#L95-L99

Reply:
@iker-barriocanal why is it resolved if its not applied?